docs/the-new-architecture/pillars-fabric-components.md

```diff
@@ -3,17 +3,37 @@ id: pillars-fabric-components
 title: Fabric Components
 ---
 
-This section contains a high-level introduction to Fabric components. It provides enough context to understand when a Fabric component is needed and how it roughly works.
-It points to the [Renderer](https://reactnative.dev/architecture/fabric-renderer) section of the [Architecture](https://reactnative.dev/architecture/overview) tab for a deep dive into the technical details.
+A Fabric Component is a UI component which is rendered on the screen using the [Fabric Renderer](https://reactnative.dev/architecture/fabric-renderer).
 
-This section must have a warning that it works only with the new architecture enabled. It points to the [migration section](../new-architecture-intro).
+Using Fabric Components instead of Native Components allows us to reap all the [benefit](./why) coming from the **New Architecture**. Specifically, we are able to leverage JSI to efficiently connect the Native UI code and the JavaScript one, and all the other features provided by the new renderer.
```

Comment:
```suggestion
Using Fabric Components instead of Native Components allows us to reap all the [benefits](./why) of the **New Architecture**. Specifically, we are able to leverage JSI to efficiently connect the Native UI and JavaScript.
```

Explanations:

* "the Native UI code and the JavaScript one" -- "code" and "one" don't match in this phrase, because "code" is [an uncountable noun](https://dictionary.cambridge.org/us/grammar/british-grammar/nouns-countable-and-uncountable). Using a countable noun like "layer" ("the Native UI layer and the JavaScript one") would be more correct, but it still sounds off to me in this phrase. "the Native UI code and the JavaScript code" would be better, as would the shorter phrase I suggested above.
* I'm wondering what the second sentence is attempting to teach the reader. As it stands, I'm not sure it adds useful information or makes the document flow better.
  * Is there something more specific we can say that "efficiently connect"? To the reader, this does not give me any new specific understanding. If not, I'd opt for removing it.
  * "all the other features provided" also does not help me understand anything new, so I'd either make it more specific or remove it entirely.
  * To make this sentence more "useful", it could be combined with the next paragraph, where we explain each step and why it's useful together (e.g., "A Fabric Component starts with a JS specification, which Codegen uses to generate some C++ code for use on the native side. This helps ensure your JS and Native interfaces have matching specifications" (first draft, I think that's possible to write better)

Reply:
Thanks for the feedback, I applied the suggestion. 

About this:
> To make this sentence more "useful", it could be combined with the next paragraph, where we explain each step and why it's useful together (e.g., "A Fabric Component starts with a JS specification, which Codegen uses to generate some C++ code for use on the native side. This helps ensure your JS and Native interfaces have matching specifications" (first draft, I think that's possible to write better)

Do you mean that we should delete the "Using Fabric Components ..." sentence or just put the two paragraph together with the suggested rewording?